monitor/seaclouds-data-collector/src/main/java/eu/seaclouds/monitor/seacloudsDc/MainDc.java

```diff
@@ -0,0 +1,62 @@
+package eu.seaclouds.monitor.seacloudsDc;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.dropwizard.Application;
+import io.dropwizard.Configuration;
+import io.dropwizard.configuration.EnvironmentVariableSubstitutor;
+import io.dropwizard.configuration.SubstitutingSourceProvider;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+
+public class MainDc extends Application<Configuration>  {
+    
+    private static final Logger logger = LoggerFactory.getLogger(MainDc.class);
+
+    private static final String CONFIG_FILE_PATH = "./config.properties";
+
+    public static void main(String[] args) throws Exception {
+
+        MainDc mainInstance = new MainDc();
+        
+        mainInstance.run(args);
+
+    }
+    
+    @Override
+    public void initialize(Bootstrap<Configuration> bootstrap) {
+        bootstrap.setConfigurationSourceProvider(
+                new SubstitutingSourceProvider(bootstrap.getConfigurationSourceProvider(),
+                        new EnvironmentVariableSubstitutor(false)
+                )
+        );
+    }
+
+    @Override
+    public void run(Configuration configuration, Environment environment) throws Exception {
```

Comment:
this is not the idiomatic way to initialize a dropwizard application.
please follow the dropwizard documentation, in particular the [getting started guide](http://www.dropwizard.io/0.9.2/docs/getting-started.html#creating-an-application-class): this will make things much easier to read